rest_framework/filters.py

```diff
@@ -127,6 +130,17 @@ def to_html(self, request, queryset, view):
         template = loader.get_template(self.template)
         return template_render(template, context)
 
+    def get_fields(self, view):
+        filter_class = getattr(view, 'filter_class', None)
```

Comment:
Possibly, tho we don't actually have a `queryset`/`model` available to us at this point, so couldn't dynamically generate the filter class if none was set explicitly.